spec/policies/time_range_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe TimeRangePolicy do
+  subject { described_class }
+
+  permissions :index?, :create?, :update?, :edit?, :destroy? do
+    it 'denies access if user is not an admin' do
+      expect(subject).not_to permit(create(:user), create(:time_range))
+    end
+
+    it 'grants access if user is an admin' do
+      expect(subject).to permit(create(:admin), create(:time_range))
+    end
```

Comment:
how about admin create admins ? :) do we block it or it's ok to allow

Reply:
It's allowed I'd say. Once one admin is set up they can create user accounts for any additional admins.
We might eventually need a super admin and we might want to restrict things like creating other admins, adding api token and editing tags / time ranges to super admins. But we'll see how it goes.